LabController/src/bkr/labcontroller/proxy.py

```diff
@@ -1,53 +1,58 @@
-
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 
+import base64
 import errno
-import os
-import sys
+import json
 import logging
-import time
-import base64
-import lxml.etree
+import os
 import re
-import json
+import shlex
 import shutil
-import tempfile
 import subprocess
+import sys
+import tempfile
+import time
+from xml.sax.saxutils import escape as xml_escape
```

Comment:
shouldn't xml.sax.. follow werkzeug....

Reply:
This is actually correct. This import is from python core, werkzeug is 3rd party. different group (separated with new line on line 20)

Reply:
I see.  TY.